### PR TITLE
Disabled drawing of creatures mouth, name and vision angles if zoomed out

### PR DESCRIPTION
**Pull request by SerayaEryn**
Disabled the drawing of the mouth, name and vision angles if zoomed out. They couldn't be seen properly anyway because they were that small.

Hopefully it improves performance a bit when zoomed out.

### CODE REVIEW

File: EvolvioColor/Creature.pde [hunk 1]
```diff
@@ -210,31 +210,12 @@ class Creature extends SoftBody {
...
-        ellipse((float)(visionOccludedX[i]*scaleUp), (float)(visionOccludedY[i]*scaleUp), 
-          2*CROSS_SIZE*scaleUp, 2*CROSS_SIZE*scaleUp);
-        stroke((float)(visionResults[i*3]), (float)(visionResults[i*3+1]), (float)(visionResults[i*3+2]));
-        strokeWeight(board.CREATURE_STROKE_WEIGHT);
-        line((float)((visionOccludedX[i]-CROSS_SIZE)*scaleUp), (float)((visionOccludedY[i]-CROSS_SIZE)*scaleUp), 
-          (float)((visionOccludedX[i]+CROSS_SIZE)*scaleUp), (float)((visionOccludedY[i]+CROSS_SIZE)*scaleUp));
-        line((float)((visionOccludedX[i]-CROSS_SIZE)*scaleUp), (float)((visionOccludedY[i]+CROSS_SIZE)*scaleUp), 
-          (float)((visionOccludedX[i]+CROSS_SIZE)*scaleUp), (float)((visionOccludedY[i]-CROSS_SIZE)*scaleUp));
-      }
+    if (showVision && camZoom > 3.5) {
```
Comment (YoYoYonnY) [inline EvolvioColor/Creature.pde]: 3.5 should probably be a constant.

File: EvolvioColor/Creature.pde [hunk 2]
```diff
@@ -249,34 +230,60 @@ class Creature extends SoftBody {
         (float)(radius*scaleUp+1+75.0/camZoom), (float)(radius*scaleUp+1+75.0/camZoom));
     }
     super.drawSoftBody(scaleUp);
+    
+    if(camZoom > 3.5)
```
Comment (YoYoYonnY) [inline EvolvioColor/Creature.pde]: ditto

File: EvolvioColor/Creature.pde [hunk 3]
```diff
@@ -249,34 +230,60 @@ class Creature extends SoftBody {
...
+      drawMouth(board, scaleUp, radius, rotation, camZoom, mouthHue);
+      
+    if (showVision && camZoom > 3.5) {
```
Comment (YoYoYonnY) [inline EvolvioColor/Creature.pde]: ditto

Probably this should all be in a single if statement instead of three.

File: EvolvioColor/Creature.pde [hunk 4]
```diff
@@ -249,34 +230,60 @@ class Creature extends SoftBody {
...
+  }
+  
+  public void drawMouth(Board board, float scaleUp, double radius, double rotation, float camZoom, double mouthHue) {
     noFill();
     strokeWeight(board.CREATURE_STROKE_WEIGHT);
     stroke(0, 0, 1);
     ellipseMode(RADIUS);
-    ellipse((float)(px*scaleUp), (float)(py*scaleUp), 
-      (float)(board.MINIMUM_SURVIVABLE_SIZE*scaleUp), (float)(board.MINIMUM_SURVIVABLE_SIZE*scaleUp));
+    ellipse((float)(px * scaleUp), (float)(py * scaleUp),
```
Comment (YoYoYonnY) [inline EvolvioColor/Creature.pde]: We really need to start using a style guide/coding standard.

Comment (izik1) [inline EvolvioColor/Creature.pde]: @YoYoYonnY I'll would love to try to help start working on that, but for now we can probably just use the default java conventions keeping in mind the auto format from processing.

**Comment by izik1** [comment]
@SerayaEryn along with what @YoYoYonnY said the whole file needs to be auto formatted via Ctrl + T the indent is one off.

**Pull request merged**